### PR TITLE
fix: removed home resources

### PR DESCRIPTION
**Pull request by tangopapatime**
# Description
Removed the home resources endpoint and removed the call to the home resources endpoint

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/page/Home.vue [hunk 1]
```diff
@@ -218,7 +218,12 @@ onMounted(async () => {
 	resourcesStore.reset(); // Project related resources saved.
 	queryStore.reset(); // Facets queries.
 
-	projects.value = ((await ProjectService.home()) ?? []).slice().reverse();
+	projects.value = ((await ProjectService.getAll()) ?? []).slice().reverse();
+
+	projects.value.forEach(async (project) => {
+		project.assets = await ProjectService.getAssets(project.id);
+		project.relatedDocuments = await getRelatedDocuments(project.id, 'xdd-covid-19');
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:225]: Could you hardcode this `xdd-covid=19` within the `getRelatedDocuments()` function directly?

**Review by YohannParis — APPROVED** [review]
Works for me.

File: packages/client/hmi-client/src/page/Home.vue (hunk 1 above)
Comment (tangopapatime) [reply packages/client/hmi-client/src/page/Home.vue:225]: yup

Comment (dvince2) [reply packages/client/hmi-client/src/page/Home.vue:225]: just a heads up here @Jami159

**Review by dvince2 — APPROVED** [review]

**Pull request merged**